apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -68,13 +68,16 @@ export const styles = {
 };
 
 // Cell formatters.
-const thumbnailFormatter = function (thumbnailUrl) {
+const thumbnailFormatter = function (thumbnailUrl, {rowData}) {
+  const projectUrl = `/projects/${rowData.type}/${rowData.channel}/`;
   thumbnailUrl = thumbnailUrl || PROJECT_DEFAULT_IMAGE;
-  return (<ImageWithStatus
-    src={thumbnailUrl}
-    width={THUMBNAIL_SIZE}
-    wrapperStyle={styles.thumbnailWrapper}
-          />);
+  return (<a style={tableLayoutStyles.link} href={projectUrl} target="_blank">
+    <ImageWithStatus
+      src={thumbnailUrl}
+      width={THUMBNAIL_SIZE}
+      wrapperStyle={styles.thumbnailWrapper}
+    />
+        </a>);
```

Comment:
nit: please fix indentation. the options are:
```
return (<a ...>
  ...
</a>);
```
or 
```
return (
  <a ...>
    ...
  </a>
);
```

Reply:
actually our style guide https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react suggests the second option is preferred.